ChangeLog.org

```diff
@@ -1,5 +1,44 @@
 * Changelog for Juvix
 
+** 0.1.1.0
+ - _Summary_:
+   + new syntax for effect handlers in Juvix.
+
+ - _Main contributions_:
+   + adds `effect` and `handler` syntaxes (also `_ via _`);
+   + removes do notation as `Prelude.»=`.
+
+*** Added Syntaxes
+#+BEGIN_SRC ocaml
+effect Print where
+  | op print : string -> ()
+  | return   : x -> string
```

Comment:
Why `return` as opposed to `pure` ?

I think the latter avoids bad associations (`return` in JS, e.g.) and is a bit more self-evident.

Reply:
Changed it to `pure`already 👍🏽